### PR TITLE
Bug: Node Table resource bar animation

### PR DESCRIPTION
**Pull request by daiyiwhale**
The resource bars on the Node Table were not always animating and displaying values properly (for example, when loading the UI in a different page and navigating to the Node page). This fix animates transform rather than width which will render properly on page changes.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

File: src/styles/components/progressbar.less [hunk 1]
```diff
@@ -20,7 +20,10 @@
   .bar {
     box-shadow: 1px 0px 0px 0px @body-background-color;
     height: 100%;
-    transition: width 600ms;
+    width: 100%;
```
Comment (jfurrow) [inline src/styles/components/progressbar.less]: Let's just :abcd: the `width` property :)

**Comment by jfurrow** [comment]
@daiyiwhale just the minor :abcd: comment then let's merge this!

**Comment by daiyiwhale** [comment]
@jfurrow ✔️

**Pull request merged**